src/backend/feed/processor.js

```diff
@@ -96,7 +96,7 @@ module.exports = async function processor(job) {
   const feed = Feed.parse(job.data);
   let articles;
   let info;
-
+  // if (!feed.isInvalid()) {
```

Comment:
Why is this commented out?

Reply:
Commented out for now as it was failing current tests, when it expects those switch res codes, it is instead always receiving 200.